backend/frontend/views.py

```diff
@@ -113,14 +121,39 @@ def can_view_user_board(logged_in_user):
         checker.prefetch_perms(viewed_user_teams)
         for view_permission in Team.PERMISSION_VIEW:
             if any(
-                (checker.has_perm(view_permission, team)
-                 for team in viewed_user_teams)
+                (checker.has_perm(view_permission, team) for team in viewed_user_teams)
             ):
                 return True
 
     return False
 
 
+def user_can_start_card(logged_in_user):
+    request = get_current_request()
+    card_id = request.resolver_match.kwargs.get("card_id")
+
+    card = get_object_or_404(AgileCard, pk=card_id)
+    card_assignees = card.assignees.all()
+
+    if (
+        (logged_in_user in card_assignees)
+        and card.can_start()
+        and not len(agile_card_reviews_outstanding(logged_in_user))
```

Comment:
From your suggestion on the "Review" btn PR, we may need these inside the actual view itself so that we can return appropriate messages. I've implemented some form of alerts on that same PR, maybe you can check them out and let me know what you think